cirq/sim/sparse_simulator_test.py

```diff
@@ -432,6 +433,41 @@ def test_compute_displays(dtype):
                                atol=1e-7)
 
 
+@pytest.mark.parametrize('dtype', [np.complex64, np.complex128])
+def test_compute_samples_displays(dtype):
+    qubits = cirq.LineQubit.range(4)
```

Comment:
You don't use qubit 0?

Suggestion: I like the style `a, b, c, d = cirq.LineQubit.range(4)`.